aten/src/ATen/native/Distance.cpp

```diff
@@ -0,0 +1,11 @@
+#include "ATen/ATen.h"
+#include "ATen/NativeFunctions.h"
+
+
+namespace at { namespace native {
+
+Tensor pairwise_distance(const Tensor& x1, const Tensor& x2, double p, double eps) {
+  auto diff = abs(x1 - x2 + eps);
+  return norm(diff, p, 1);
```

Comment:
Nit: norm already takes the abs of it's input, so you can rewrite pairwise_distance as `return norm(x1 - x2 + eps, p, 1)`